addons/binding/org.openhab.binding.rfxcom/src/main/java/org/openhab/binding/rfxcom/internal/messages/RFXComInterfaceControlMessage.java

```diff
@@ -0,0 +1,126 @@
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.rfxcom.internal.messages;
+
+import java.util.List;
+
+import org.eclipse.smarthome.core.types.State;
+import org.eclipse.smarthome.core.types.Type;
+import org.openhab.binding.rfxcom.internal.config.RFXComBridgeConfiguration;
+import org.openhab.binding.rfxcom.RFXComValueSelector;
+import org.openhab.binding.rfxcom.internal.exceptions.RFXComException;
+
+/**
+ * RFXCOM data class for control message.
+ *
+ * @author Pauli Anttila - Initial contribution
+ */
+public class RFXComInterfaceControlMessage extends RFXComBaseMessage {
+
+    private byte[] data = new byte[14];
+
+
+    public RFXComInterfaceControlMessage(RFXComInterfaceMessage.TransceiverType transceiverType, RFXComBridgeConfiguration configuration) {
+        data[0] = 0x0D;
+        data[1] = RFXComBaseMessage.PacketType.INTERFACE_CONTROL.toByte();
+        data[2] = 0;
+        data[3] = 2;
+        data[4] = RFXComInterfaceMessage.Commands.SET_MODE.toByte();
+        data[5] = transceiverType.toByte();
+        data[6] = 0;
+
+        data[7] = (byte)(
+              (configuration.enableUndecoded        ? 0x80 : 0x00)
+            | (configuration.enableImagintronixOpus ? 0x40 : 0x00)
+            | (configuration.enableByronSX          ? 0x20 : 0x00)
+            | (configuration.enableRSL              ? 0x10 : 0x00)
+            | (configuration.enableLighting4        ? 0x08 : 0x00)
+            | (configuration.enableFineOffsetViking ? 0x04 : 0x00)
+            | (configuration.enableRubicson         ? 0x02 : 0x00)
+            | (configuration.enableAEBlyss          ? 0x01 : 0x00)
+        );
+
+        data[8] = (byte)(
+              (configuration.enableBlindsT1T2T3T4   ? 0x80 : 0x00)
+            | (configuration.enableBlindsT0         ? 0x40 : 0x00)
+            | (configuration.enableProGuard         ? 0x20 : 0x00)
+         // | (configuration.enableFS20Packets      ? 0x10 : 0x00)
+            | (configuration.enableLaCrosse         ? 0x08 : 0x00)
+            | (configuration.enableHidekiUPM        ? 0x04 : 0x00)
+            | (configuration.enableADLightwaveRF    ? 0x02 : 0x00)
+            | (configuration.enableMertik           ? 0x01 : 0x00)
+        );
+
+        data[9] = (byte)(
+              (configuration.enableVisonic          ? 0x80 : 0x00)
+            | (configuration.enableATI              ? 0x40 : 0x00)
+            | (configuration.enableOregonScientific ? 0x20 : 0x00)
+            | (configuration.enableMeiantech        ? 0x10 : 0x00)
+            | (configuration.enableHomeEasyEU       ? 0x08 : 0x00)
+            | (configuration.enableAC               ? 0x04 : 0x00)
+            | (configuration.enableARC              ? 0x02 : 0x00)
+            | (configuration.enableX10              ? 0x01 : 0x00)
+        );
+
+        data[10] = 0;
```

Comment:
Homeconfort is a specific message type but we can already receive it via undecoded messages but it requires both (undecoded and homeconfort) of them to be enabled.

Keeloq is indeed not supported at all, it can only be processed as a Security2 message which is not supported, but lets include already the one including support the message type will forget it ;-)